src/bundler.ts

```diff
@@ -469,7 +472,13 @@ export class Bundler {
     if (!bundle.bundle.files.has(bundle.url)) {
       return this._analyzeContents(bundle.url, '');
     }
-    const document = await this.analyzer.analyze(bundle.url);
+    const analyzer = await this.analyzer;
+    const analysis = await analyzer.analyze([bundle.url]);
```

Comment:
do we not support analyzing single files anymore? `analyzer.analyze(bundle.url)` should be fine here